client/my-sites/earn/memberships/index.jsx

```diff
@@ -217,13 +217,13 @@ class MembershipsSection extends Component {
 				{ Object.values( this.props.subscribers ).length === 0 && (
 					<Card>
 						{ this.props.translate(
-							"You haven't added any subscribers. {{a}}Learn more{{/a}} about recurring payments.",
+							"You haven't added any subscribers. {{a}}Learn more{{/a}} about payments.",
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.